packages/@aws-cdk/assert/test/assertions.test.ts

```diff
@@ -266,34 +266,34 @@ passingExample('expect <stack> to count resources like props - expected no resou
   cdkExpect(synthStack).to(countResourcesLike(resourceType, 0, { parentId: 456 }));
 });
 
-failingExample('expect <synthStack> to count resources like props - more than expected', () => {
```

Comment:
Please avoid modifying existing tests and instead add new tests

Reply:
Are you saying it would it be preferable to completely remove the previous `failingExample`s and create wholly new `passingExample`s that are different enough to not trip up GitHub's diff?

The diff is a bit misleading - all that was changed is that the previous `failingExample`s are now `passingExample`s; incomplete child properties & child properties provided in a different order than they appear in the `resource.Properties` object now match so long as the `resource.Properties` object is a superset of the properties provided to `countResourcesLike`.

Reply:
Oh I get it. So please revert the reordering so that the diff demonstrates that existing tests were not touched. It is usually not a good practice to reorder code as part of another change because it is hard for reviewers to "trust" that things did not change.

If you believe we should reorder the tests, you are more than welcome to submit a separate PR that only includes the reorganization.